.changeset/swift-penguins-fail.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
@kate-masaki 
Please only include "@yamada-ui/radio": major and "@yamada-ui/theme": major in the changeset.